src/components/swapv2/TokenInfoV2.tsx

```diff
@@ -26,33 +26,29 @@ const checkTokenDescription = ({
   chainId: ChainId | undefined
 }) => {
   // hard code pair description for SEO
-  const rs1 = JSON.parse(JSON.stringify(tokenInfo1))
-  const rs2 = JSON.parse(JSON.stringify(tokenInfo2))
-  let isHardCode = false
+  const rs1: TokenInfo = JSON.parse(JSON.stringify(tokenInfo1))
```

Comment:
This is not a good way to deep-clone an object. Link: https://stackoverflow.com/a/5344074/6812545

We should use `immer` here.

Reply:
ah, after review. I see there is no need to deep copy.  I update it to 
<img width="732" alt="image" src="https://user-images.githubusercontent.com/33005392/178178086-9e7e2922-1762-48fc-8586-15426dcd0385.png">
thanks for you suggestion.

Reply:
Yeah, even in that case, you can consider using immer. It'd be much more readable